src/pydvl/value/oob/oob.py

```diff
@@ -0,0 +1,139 @@
+"""
+## References
+
+[^1]: <a name="kwon_data_2023"></a>Kwon et al. [Data-OOB: Out-of-bag Estimate as a Simple and Efficient Data Value]
+(https://proceedings.mlr.press/v202/kwon23e.html). In: Published at ICML 2023
+
+"""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from typing import TypeVar
+
+import numpy as np
+from numpy.typing import NDArray
+from sklearn.base import is_classifier, is_regressor
+from sklearn.ensemble import BaggingClassifier, BaggingRegressor
+
+from pydvl.utils import Utility, maybe_progress
+from pydvl.value.result import ValuationResult
+
+__all__ = ["compute_data_oob"]
+
+T = TypeVar("T", bound=np.number)
+
+
+def compute_data_oob(
+    u: Utility,
+    n_est: int = 10,
+    max_samples: float = 0.8,
+    n_jobs: int = None,
+    loss: Callable = None,
+    *,
+    progress: bool = True,
```

Comment:
For almost all methods we set this to False be default to avoid cluttering the terminal:

```suggestion
    progress: bool = False,
```